cmd/policy-attributeNamespaces.go

```diff
@@ -155,6 +159,108 @@ func policy_updateAttributeNamespace(cmd *cobra.Command, args []string) {
 	HandleSuccess(cmd, id, t, ns)
 }
 
+func policy_unsafeDeleteAttributeNamespace(cmd *cobra.Command, args []string) {
+	h := NewHandler(cmd)
+	defer h.Close()
+
+	flagHelper := cli.NewFlagHelper(cmd)
+	id := flagHelper.GetRequiredString("id")
+
+	ns, err := h.GetNamespace(id)
+	if err != nil {
+		errMsg := fmt.Sprintf("Failed to find namespace (%s)", id)
+		cli.ExitWithError(errMsg, err)
+	}
+
+	if !forceUnsafe {
+		cli.ConfirmTextInput(cli.ActionDelete, "namespace", cli.InputNameFQN, ns.GetFqn())
+	}
+
+	if err := h.UnsafeDeleteNamespace(id, ns.GetFqn()); err != nil {
+		errMsg := fmt.Sprintf("Failed to delete namespace (%s)", id)
+		cli.ExitWithError(errMsg, err)
+	}
```

Comment:
Instead of doing a client side check could we send the user input to the server for validation? This way we don't have risk of defects and accidental deletion in the client. 

If they are forcing it then we can prefill with the known FQN

Reply:
We currently require FQNs in the backend (which are validated) for only `delete`; `update` and `reactivate` do _not_ require FQNs in the unsafe request. This confirmation flow seemed beneficial because the user wouldn't have to first print JSON in a separate command then copy and/or remember the FQN to type in the confirmation screen if we do a roundtrip in the client. We have some options. 

1. We could add FQNs to `update` and `reactivate` and make them required fields to have the serverside validation for all unsafe requests. Then we could pass the user's interactive entry directly or retrieve it ourselves for `--force`.
2. We could do 1 _and_ move every fqn entry to a flag `--fqn` then make confirmation a `y/n` like deactivation for consistency instead of typing in the FQN values interactively.
3. We could not do 1, leave FQNs off the `update` and `reactivate` requests, and do a `y/n` confirmation for them and an FQN interactive entry where the value is passed directly into the request (without verification clientside). 

To me, it doesn't seem like doing an extra validation for UX is very likely get us in a troublesome loop as long as validation happens in the server for security/DX of the API.

Reply:
Okay let's resolve this later. Thanks for the context.

Reply:
https://github.com/opentdf/otdfctl/issues/222